public/app/features/provisioning/JobStatus.tsx

```diff
@@ -16,38 +16,62 @@ export function JobStatus({ name, onStatusChange }: JobStatusProps) {
   const jobQuery = useListJobQuery({ watch: true, fieldSelector: `metadata.name=${name}` });
   const job = jobQuery.data?.items?.[0];
   const hasNotifiedSuccess = useRef(false);
-  const isSuccess = job?.status?.state === 'success';
 
   useEffect(() => {
-    if (isSuccess && onStatusChange && !hasNotifiedSuccess.current) {
+    if (onStatusChange && !hasNotifiedSuccess.current) {
       hasNotifiedSuccess.current = true;
       onStatusChange(true);
     }
-  }, [isSuccess, onStatusChange]);
+  }, [onStatusChange]);
 
   if (jobQuery.isLoading || !job) {
     return (
       <Stack direction="column" alignItems="center" gap={2}>
         <Spinner />
-        <Text>Starting migration...</Text>
+        <Text>Starting...</Text>
       </Stack>
     );
   }
 
+  const status = () => {
+    switch (job.status?.state) {
+      case 'success':
+        return <Alert severity="success" title="Migration succesful" />;
+      case 'error':
+        return (
+          <Alert severity="error" title="error running job">
+            {job.status.message}
+          </Alert>
+        );
+    }
+    return (
+      <Stack direction="row">
+        <Spinner />
+        <Text element="p" weight="medium">
+          {job.status?.message ?? ''}
```

Comment:
I noticed there's no message sometimes, should we show some placeholder?

Reply:
umm, that's weird. Is it at the start or during? It shouldn't be the case.